website/docs/docs/collaborate/govern/model-contracts.md

```diff
@@ -76,9 +76,18 @@ When building a model with a defined contract, dbt will do two things differentl
 
 ## FAQs
 
+### Where are contracts supported?
```

Comment:
non blocker but this section almost feels like it should be on the top as part of the intro. not really a prerequisites

Reply:
good call!